lib/database.js

```diff
@@ -47,8 +47,15 @@ function Database(filenameGiven, options) {
 	if (nativeBindingPath == null) {
 		addon = DEFAULT_ADDON || (DEFAULT_ADDON = require('bindings')('better_sqlite3.node'));
 	} else {
-		addon = require(path.resolve(nativeBindingPath).replace(/(\.node)?$/, '.node'));
+		// If webpack is used to build the application, we must use `__non_webpack_require__`.
+		const requireFunc =
+			typeof __webpack_require__ === 'function'
+				? __non_webpack_require__
```

Comment:
Did you mean to check two different things? I would expect something like 

```
typeof __non_webpack_require__ === 'function' ? __non_webpack_require__ : require
```

Reply:
To me it looks fine, the ternary operation also incluse line 54, guess the formatting is a bit off compared to the rest of the code.

Reply:
Sorry, I guess I wasn't clear. 

They test that `__webpack_require__` is a function, but then _hope_ that **`__non`**`_webpack_require__` exists and is a function. They should be both testing and using the same thing.

Reply:
Ah yes, my brain skipped the "\_\_non__" part. But then everything _should_ be fine. Both variables are only set / present if webpack is used. So if there is a \_\_webpack_require__ defined, by convention there is also the \_\_non__ version.

Reply:
> So if there is a __webpack_require__ defined, by convention there is also the __non__ version.

Currently. I've been burned in more ways than I can count by webpack changing stuff seemingly arbitrarily between (even minor) versions.

Reply:
Okay guys, I fixed it. Thank you